content/docs/guides/crosswalk/kubernetes/aws/02-kubernetes-stacks/01-cluster-configuration.md

```diff
@@ -0,0 +1,672 @@
+---
+
+title: Kubernetes Cluster
+
+aliases: ["/docs/guides/crosswalk/kubernetes/aws/cluster/"]
+---
+
+Manage the control plane, worker nodes, RBAC, and defaults.
```

Comment:
Can we improve the prose here, so it reads like documentation? For example, this could be a paragraph might be

> In order to run container workloads, you will need a cluster. It is possible to provision and manage a cluster manually on AWS, however the managed offering, Elastic Kubernetes Service (EKS), offers an easier way to get up and running. The primary steps you will need to follow include:

Not saying this is perfect, but it reads more like documentation that's going to walk you through something.

Reply:
Done